clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/provider/AwsProvider.groovy

```diff
@@ -134,4 +130,34 @@ class AwsProvider extends AgentSchedulerAware implements SearchableProvider {
       return cacheView.getAll(INSTANCES.ns, possibleInstanceIdentifiers)*.id
     }
   }
+
+  // Eureka provider support
+  private Map<String, String> accounIdToAccountNameLookup = [:]
+
+  @Override
+  Boolean isProviderForEurekaRecord(Map<String, Object> attributes) {
+    attributes.containsKey('accountId') && attributes.get('titusTaskId') == null
```

Comment:
can we do something where this class doesn't have to know about `titusTaskId`? an `@Ordered` set of `EurekaAwareProvider` and the first one that returns true for `isProviderForEurekaRecord` wins?